packages/aws-cdk-lib/aws-logs/lib/log-retention.ts

```diff
@@ -73,13 +80,14 @@ export interface LogRetentionRetryOptions {
  * Log group can be created in the region that is different from stack region by
  * specifying `logGroupRegion`
  */
-export class LogRetention extends Construct {
-
+export class LogRetention extends Construct implements cdk.ITaggable {
   /**
    * The ARN of the LogGroup.
    */
   public readonly logGroupArn: string;
 
+  public readonly tags: cdk.TagManager = new cdk.TagManager(cdk.TagType.KEY_VALUE, 'Custom::LogRetention');
```

Comment:
I'm thinking we might want to change the type here from `Custom::LogRetention`
to `AWS::Logs::LogGroup` since we are tagging the log group and not the custom
resource itself. It would also allow the user to more intuitively filter for log
groups when tagging.

Reply:
I've updated this to be `AWS::Logs::LogGroup`